src/components/TodoInfo/TodoInfo.tsx

```diff
@@ -1 +1,30 @@
-export const TodoInfo = () => {};
+import React from 'react';
+import { Todo } from '../../types/Todo';
+import { UserInfo } from '../UserInfo';
+
+type Props = {
+  todo: Todo
+};
+
+export const TodoInfo: React.FC<Props> = ({ todo }) => {
+  const {
+    completed,
+    title,
+    user,
+    id,
+  } = todo;
+
+  return (
+    <article
+      key={id}
+      data-id={id}
+      className={`TodoInfo ${completed
+        ? 'TodoInfo--completed'
+        : ''}`}
```

Comment:
You can use a classnames library to add classes with condition, it will be better